edgexfoundry-holding/device-onvif-camera
# feat: Update build script

- Change Dockerfile golang version from golang:1.16-alpine3.14 to golang:1.17-alpine3.15
- Update go.mod to use go 1.17
- The package name should be edgexfoundry

Closes #4

Signed-off-by: bruce <weichou1229@gmail.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/IOTechSystems/device-onvif-camera/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **not impact the test**
- [ ] I have opened a PR for the related docs change (if not, why?) **the doc is modified in this PR**
  <link to docs PR>

## Testing Instructions

1. Run core services and device-onvif-camera to verify the functionality
2. Run `make test` and `make docker`

## New Dependency Instructions (If applicable)
